LeaderboardBackend/Controllers/CategoriesController.cs

```diff
@@ -30,6 +30,26 @@ public async Task<ActionResult<CategoryViewModel>> GetCategory([FromRoute] long
         return Ok(CategoryViewModel.MapFrom(category));
     }
 
+    [AllowAnonymous]
+    [HttpGet("api/leaderboard/{id:long}/category")]
+    [SwaggerOperation("Gets a Category of Leaderboard `id` by its slug. Will not return deleted Categories.", OperationId = "getCategoryBySlug")]
+    [SwaggerResponse(200)]
+    [SwaggerResponse(404)]
```

Comment:
Specify that this returns a Problem Details.

Reply:
There's two conflicting things I've found now that you brought this up:
1. Looking at e.g. the CreateCategory action; we modify its `NotFound` call with a custom `ProblemDetails`
2. This clashes with GetLeaderboardBySlug, which is what I based GetCategoryBySlug on
Do we want to follow the former's pattern? I think we do don't we? And if that's the case; it'll mean test edits, too. Though I kinda feel like this change will be pointless.